frontend/src/components/dashboard/AvatarRow.vue

```diff
@@ -18,14 +18,17 @@ export default {
   components: { UserAvatar },
   props: {
     campCollaborations: { type: Array, default: () => [] },
-    size: { type: [Number, String], default: 20 },
+    maxSize: { type: [Number, String], default: 20 },
   },
+  data: () => ({
+    maxHeight: 1000,
+  }),
   computed: {
+    size() {
+      return Number(this.maxSize) < this.maxHeight ? Number(this.maxSize) : this.maxHeight
```

Comment:
Correct, I've implemented this for the picasso. I think, for big screens/wide scheduleEntry widths the avatars could be used instead of the text.
But I've decided to do this in a next PR.
But maybe I should move it to a more general place.

Reply:
Okay makes sense. But I would [KISS](https://en.wikipedia.org/wiki/KISS_principle) as long as you don't actually reuse this in another place. You cannot possibly know how you will implement that future picasso feature. The leftover components I noticed in this PR are living proof of that. You can still move the component and make it more complicated once you actually reuse this component in the Picasso.